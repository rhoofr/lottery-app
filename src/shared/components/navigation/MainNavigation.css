.main-navigation__menu-btn {
  width: 3rem;
  height: 3rem;
  background: transparent;
  border: none;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  margin-right: 2rem;
  cursor: pointer;
}

.main-navigation__menu-btn span {
  display: block;
  width: 2.7rem;
  height: 2.5px;
  background: white;
}

.main-navigation__title {
  margin-left: 4rem;
  color: white;
  padding-bottom: 0.9rem;
}

.main-navigation__title a {
  text-decoration: none;
  color: white;
}

.main-navigation__header-nav {
  display: none;
}

.main-navigation__drawer-nav {
  height: 100%;
}

@media (min-width: 1200px) {
  .main-navigation__menu-btn {
    display: none;
  }

  .main-navigation__header-nav {
    display: block;
    margin-right: 4rem;
  }
}
