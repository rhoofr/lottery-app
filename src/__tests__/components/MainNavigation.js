import React from 'react';
import { shallow } from 'enzyme';
import MainNavigation from '../../shared/components/navigation/MainNavigation';

describe('--- MainNavigation snapshot ---', () => {
  test('should render Main Navigation correctly.', () => {
    const wrapper = shallow(<MainNavigation />);
    expect(wrapper).toMatchSnapshot();
  });
});
