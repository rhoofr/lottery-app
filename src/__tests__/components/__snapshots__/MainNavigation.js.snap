// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--- MainNavigation snapshot --- should render Main Navigation correctly. 1`] = `
<Fragment>
  <SideDrawer
    onClick={[Function]}
    show={false}
  >
    <nav
      className="main-navigation__drawer-nav"
    >
      <NavLinks />
    </nav>
  </SideDrawer>
  <MainHeader>
    <button
      className="main-navigation__menu-btn"
      onClick={[Function]}
    >
      <span />
      <span />
      <span />
    </button>
    <h1
      className="main-navigation__title"
    >
      <Link
        to="/"
      >
        Lottery - App
      </Link>
    </h1>
    <nav
      className="main-navigation__header-nav"
    >
      <NavLinks />
    </nav>
  </MainHeader>
</Fragment>
`;
